site/content/contribute/getting-started/core-committers.md

```diff
@@ -49,6 +49,15 @@ Below is the list of core committers working on Mattermost:
 - **Sudheer Timmaraju**
   - @sudheerdev on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@sudheerdev) and [@sudheerDev](https://github.com/sudheerDev) on GitHub
   - Dev areas: Web App, React, Redux, Performance
+- **Daniel Schalla**
+  - @dschalla on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@dschalla) and [@dschalla](https://github.com/dschalla) on GitHub
+  - Dev areas: Security, Plugins, Integrations, Performance
+- **Hyeseong Kim**
+  - @cometkim on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@cometkim) and [@cometkim](https://github.com/cometkim) on GitHub
+  - Dev areas: Web App, React, Redux, REST API
+- **Hanzei**
+  - @hanzei on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@hanzei) and [@hanzei](https://github.com/hanzei) on GitHub
+  - Dev areas: Plugins, Integrations, Matterpoll
```

Comment:
Please remove `Matterpoll`. Its a personal project and not an "official" Mattermost project.

Would you please add `Security`? I know I did't a contribution related to this, but its a topic which interests me.

Reply:
For sure, I'll update it

Reply:
https://github.com/mattermost/mattermost-developer-documentation/commit/da9974ad7d203da0bdf0b0541eb430b9746d6426